clang/test/Driver/sycl-offload-with-split.c

```diff
@@ -245,7 +245,7 @@
 // CHK-PHASE-MULTI-TARG: 3: input, "[[INPUT]]", c++, (device-sycl)
 // CHK-PHASE-MULTI-TARG: 4: preprocessor, {3}, c++-cpp-output, (device-sycl)
 // CHK-PHASE-MULTI-TARG: 5: compiler, {4}, ir, (device-sycl)
-// CHK-PHASE-MULTI-TARG: 6: offload, "host-sycl (x86_64-unknown-linux-gnu)" {2}, "device-sycl (spir64-unknown-unknown)" {5}, c++-cpp-output
+// CHK-PHASE-MULTI-TARG: 6: offload, "host-sycl (x86_64-unknown-linux-gnu)" {2}, "device-sycl (spir64_gen-unknown-unknown)" {5}, c++-cpp-output
```

Comment:
Note to self & other reviewers: this change concerned me at first, as I've doubted the deterministic ordering of the target toolchains after the map introduction, but then realized that usage of `MapVector` for toolchain/arch mapping [ensures that](https://llvm.org/doxygen/classllvm_1_1MapVector.html#details).